docs/deployment.md

```diff
@@ -0,0 +1,98 @@
+# Deploying OBMd
+
+## Download OBMd
+
+1. Download the OBMd binary from github. This downloads v0.2 of obmd.
+
+`$ wget https://github.com/CCI-MOC/obmd/releases/download/v0.2/obmd`
+
+Alternatively, you can clone the source code and build the binary yourself.
+
+2. Make it executable and place it in `/bin`.
+
+```
+$ sudo chmod +x obmd
+$ sudo mv obmd /bin/
+```
+
+3. If you decide to move obmd to a different directory then make sure to update
+the obmd service file.
+
+## The configuration file
+
+1. Create a configuration file for obmd `/etc/obmd/config.json`.
```

Comment:
Might want to point the user to the README for more complete docs on the config file.